MIGRATING.md

```diff
@@ -485,9 +489,9 @@ major releases of `cosmwasm`. Note that you can also view the
 
 - Add `MessageInfo` as separate arg after `Env` for `init`, `handle`, `migrate`.
   Add `Env` arg to `query`. Use `info.sender` instead of `env.message.sender`
-  and `info.sent_funds` rather than `env.message.sent_funds`. Just changing the
```

Comment:
I assume you didn't want this change? We didn't make that change in 0.13 but only in 0.14 (hence the entry above)

Reply:
Will revert.